opencti-platform/opencti-graphql/src/domain/feed.ts

```diff
@@ -73,7 +75,15 @@ export const editFeed = async (context: AuthContext, user: AuthUser, id: string,
   return findById(context, user, id);
 };
 export const findAll = (context: AuthContext, user: AuthUser, opts: QueryFeedsArgs) => {
-  return listEntitiesPaginated<StoreEntityFeed>(context, user, [ENTITY_TYPE_FEED], opts);
+  if (user) {
+    const options = { ...opts, includeAuthorities: true };
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
What was the issue ?

Reply:
Some typescript errors. I manage to find the solution.